sc-cli.go

```diff
@@ -0,0 +1,9 @@
+package main
```

Comment:
Ok, so we could do that :) Question is do we want to. As per cobra user documentation, they suggest a layout that the main binary is NOT in the same directory as the commands, and they suggest:
 sc-cli.go
 /cmd
    commands go here

So, I really don't like lumping everything into the cmd directory and having them in the main package without a good reason. Anybody else who comes into the project with cobra might be surprised by this, I also think there are tools that assume this structure. 
So, I see two viable options:
  - leave the sc-cli.go at the top, you don't like that I know :)
 - introduce something like cli/ directory and have sc-cli.go there, with cmd/<commands> underneath that.

Are you fine with the second option? Seems like it would allow us to conform to suggested cobra structure, and move the sc-cli.go out of the root which I understood was your biggest concern.

Thoughts?